web/src/components/ShelterFilterBar.tsx

```diff
@@ -41,6 +42,15 @@ const ShelterFilterBar: React.FC<ShelterFilterBarProps> = ({
         updateSearchFilter={updateSearchFilter}
         name='beds'
       />
+      <FacetInput
+        value={filter.city ?? ''}
+        icon={cityIcon}
+        altTag={t('facetCity')}
+        placeholder={t('facetCity')}
+        updateSearchFilter={updateSearchFilter}
+        type='string'
```

Comment:
I don't think `string` is a valid input type, see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input#input_types. I think the correct one would be `text`. To avoid this kind of mistakes you should add a more specific type to the `type` prop which only accepts valid input types.